README_v2.md

```diff
@@ -0,0 +1,176 @@
+Please refer to the [Solr Operator](https://solr.apache.org/operator) home for details, tutorials and instructions.
+
+## Building Solr from Source
+FIXME --> is this where we get JDK from??   What about OpenJDK?
+Download the Java 11 JDK (Java Development Kit) or later from https://jdk.java.net/
+You will need the JDK installed, and the $JAVA_HOME/bin (Windows: %JAVA_HOME%\bin)
+folder included on your command path. To test this, issue a "java -version" command
+from your shell (command prompt) and verify that the Java version is 11 or later.
+
+Download the Apache Solr distribution, from http://solr.apache.org/.  FIXME link??
+Unzip the distribution to a folder of your choice, e.g. C:\solr or ~/solr
+Alternately, you can obtain a copy of the latest Apache Solr source code
+directly from the Git repository:
+
+<https://solr.apache.org/community.html#version-control>
+
+As of 9.0, Solr uses [Gradle](https://gradle.org/) as the build
+system.  Navigate to the root of your source tree folder and issue the `./gradlew tasks`
+command to see the available options for building, testing, and packaging Solr.
+
+`./gradlew assemble` will create a Solr executable.
+cd to `./solr/packaging/build/solr-9.0.0-SNAPSHOT` and run the `bin/solr` script
+to start Solr.
+
+NOTE: `gradlew` is the "Gradle Wrapper" and will automatically download and
+start using the correct version of Gradle for Solr.
+
+NOTE: `./gradlew help` will print a list of high-level tasks. There are also a
+number of plain-text files in <source folder root>/help.
+
+The first time you run Gradle, it will create a file "gradle.properties" that
+contains machine-specific settings. Normally you can use this file as-is, but it
+can be modified if necessary.
+
+Note as well that the gradle build does not create or copy binaries throughout the
+source repository so you need to switch to the packaging output folder `./solr/packaging/build`;
+the rest of the instructions below remain identical. The packaging directory
+is rewritten on each build.
+
+If you want to build the documentation, type `./gradlew -p solr documentation`.
+
+`./gradlew check` will assemble Solr and run all validation tasks unit tests.
+
+Lastly, there is developer oriented documentation in `./dev-docs/README.adoc` that
+you may find useful in working with Solr.
+
+
+### Gradle build and IDE support
+
+- *IntelliJ* - IntelliJ idea can import the project out of the box.
+               Code formatting conventions should be manually adjusted.
+- *Eclipse*  - Not tested.
+- *Netbeans* - Not tested.
+
+## Contributing
+
+Please review the [Contributing to Solr Guide](https://cwiki.apache.org/confluence/display/solr/HowToContribute)
+for information on contributing.
+
+## Discussion and Support
+
+- [Mailing Lists](https://solr.apache.org/community.html#mailing-lists-chat)
+- [Issue Tracker (JIRA)](https://issues.apache.org/jira/browse/SOLR)
+- IRC: `#solr` and `#solr-dev` on freenode.net
```

Comment:
I think we have officially moved from freenode to https://libera.chat/, but check with @chatman who worked with the cutover and the slack bridge??